plugins/catalog/src/components/SystemDiagramCard/SystemDiagramCard.tsx

```diff
@@ -157,7 +158,7 @@ export function SystemDiagramCard() {
         kind: ['Component', 'API', 'Resource', 'System', 'Domain'],
         'spec.system': [
           currentSystemName,
-          `${ENTITY_DEFAULT_NAMESPACE}/${currentSystemName}`,
+          `${ref.namespace || 'Current Namespace'}/${currentSystemName}`,
```

Comment:
We should be able to use `entity.metadata.namespace` here instead, and then fall back to the default namespace instead.

Reply:
i got it! i will make the change